src/domain/cart/CartPage.tsx

```diff
@@ -1,5 +1,39 @@
+import { useSelector } from "react-redux";
+import "./style.css";
+import { RootState } from "../../store/store";
+import CartCard from "../../components/cartCard/CartCard";
+import { CartItem } from "./types";
+
 export const CartPage = (): JSX.Element => {
+  const cartdata = useSelector((state: RootState) => state.cart.cart);
```

Comment:
Like above, create selectors inside slice files and use them here.

Reply:
sure...